<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TorreDeHanoi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_Pino_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>21</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Pino_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>21</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Pino_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>21</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pino 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pino 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pino 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pino de Origem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pino de Destino</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Pino_De_Origem">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Pino_De_Destino">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Trocar_Disco">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Trocar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Restart">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Jogadas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_quantidade_De_Jogadas">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>10</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Numero de Discos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_QuantidadeDeDiscos">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>12</y>
      <width>57</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CriarJogo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>11</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>textEdit_Pino_1</zorder>
   <zorder>textEdit_Pino_2</zorder>
   <zorder>textEdit_Pino_3</zorder>
   <zorder>line</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit_Pino_De_Origem</zorder>
   <zorder>lineEdit_Pino_De_Destino</zorder>
   <zorder>pushButton_Trocar_Disco</zorder>
   <zorder>pushButton_Restart</zorder>
   <zorder>label_7</zorder>
   <zorder>label_quantidade_De_Jogadas</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>257</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
